### PR TITLE
This one is even more pep8 clean up, based on pycharm, there are 0 violation.

### PR DESCRIPTION
**Pull request by lsetiawan**

### CODE REVIEW

**Comment by lsetiawan** [comment]
Hold on this, I'm going to go through the travis log and fix each error.

**Comment by lsetiawan** [comment]
@ocefpaf Is it okay to increase the line length a bit? There are some variables that have long names and really hard to break up.

**Comment by ocefpaf** [comment]
> @ocefpaf Is it okay to increase the line length a bit? There are some variables that have long names and really hard to break up.

Sure, but ideally you can mark the long line that you want to remain as-is with a comment `# noqa`. That will inform to linter to leave that line alone.

**Comment by lsetiawan** [comment]
@ocefpaf Great! Thanks for letting me know :)

**Comment by lsetiawan** [comment]
@ocefpaf @miguelcleon All checks are passed now. Though I hope that there aren't a lot of bugs, since lots of the clean up was done by PyCharm. So far that I have tested, they look good. Thanks.

**Comment by miguelcleon** [comment]
Sorry, I got pulled onto another project but I should be back on this soon. It's great that it passed travis, hopefully it still works.

**Comment by ocefpaf** [comment]
I would be nice to get this one merged before any other change to avoid forcing people to rebase. This PR touches almost all the files!

**Comment by miguelcleon** [comment]
I'm looking at it now.

**Comment by miguelcleon** [comment]
the base map MapBox_RunBikeHike is not loading for me at all, the ESRI Basemaps load fine. The OSM Run Bike Hike loads for you?

**Comment by lsetiawan** [comment]
Ah, sorry, here's the access token to be added in settings.py pk.eyJ1IjoibHNldGlhd2FuIiwiYSI6ImNpbjI3M2UzNzBiZzh2OWtrZGlzZ2FhaG8ifQ.tkoR6uJikfFpOq4jfsk02w

**Comment by miguelcleon** [comment]
Ok, yes that fixed it, thanks. Looking at graphs now. when I click on a time series, I get a tiny graph with a proper height but that is very narrow, probably a width setting isn't getting set.

File: templatesAndSettings/templates/chart2.html [hunk 1]
```diff
@@ -157,21 +157,21 @@
 {% if not useSamplingFeature and not useDataset %}
 Method- {{featureActionMethod}}
 {% endif %}
-</h1></p>
+</h1>
     <div id='content-main'> {% csrf_token %}
 	 </div>
-	<div id='chartid' class="chart" style="height:75%; width:100 margin: 0 auto"></div>
+	<div id='chartid' class="chart" style="height:75%; width:100px; margin: 0 auto;"></div>
```
Comment (miguelcleon) [inline templatesAndSettings/templates/chart2.html:163]: width shouldn't be 100px should be 100%

**Comment by miguelcleon** [comment]
getting an error when selecting different datasets from the map 
local variable 'externalidentifiers' referenced before assignment
`\ODM2AdminPep8\ODM2CZOData\views.py in web_map
        'externalidentifiers': externalidentifiers,`

nevermind I think this error existed already and is not your fault.

File: ODM2CZOData/NewProcessinglevelForTimeSeries.py [hunk 2]
```diff
@@ -1,64 +1,82 @@
...
-timeRangesToRemove.append([datetime.strptime('2014-03-29 21:30', '%Y-%m-%d %H:%M'),datetime.strptime('2014-04-03  10:00', '%Y-%m-%d %H:%M')]) # noqa
-timeRangesToRemove.append([datetime.strptime('2014-06-16 20:00', '%Y-%m-%d %H:%M'),datetime.strptime('2014-06-19  3:30', '%Y-%m-%d %H:%M')]) # noqa
-timeRangesToRemove.append([datetime.strptime('2014-06-19 20:45', '%Y-%m-%d %H:%M'),datetime.strptime('2014-06-20  11:30', '%Y-%m-%d %H:%M')]) # noqa
-timeRangesToRemove.append([datetime.strptime('2014-06-21 15:30', '%Y-%m-%d %H:%M'),datetime.strptime('2014-06-30  18:30', '%Y-%m-%d %H:%M')]) # noqa
-timeRangesToRemove.append([datetime.strptime('2014-07-01 4:00', '%Y-%m-%d %H:%M'),datetime.strptime('2014-07-05  17:45', '%Y-%m-%d %H:%M')]) # noqa
-timeRangesToRemove.append([datetime.strptime('2014-07-26 15:00', '%Y-%m-%d %H:%M'),datetime.strptime('2014-08-01  1:15', '%Y-%m-%d %H:%M')]) # noqa
-timeRangesToRemove.append([datetime.strptime('2015-04-24 15:00', '%Y-%m-%d %H:%M'),datetime.strptime('2015-05-08  11:15', '%Y-%m-%d %H:%M')]) # noqa
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "templatesAndSettings.settings")
```
Comment (miguelcleon) [inline ODM2CZOData/NewProcessinglevelForTimeSeries.py:10]: This won't actually work; Maybe this needs to go in a different project. 

If you try to run it as a seperate Django ORM script which is what it is supposed to be, it needs the settings at the very start of the file.

**Comment by miguelcleon** [comment]
I went ahead and merged and then issued a commit, which fixes the graph width issue and the externalidentifiers issue (when selecting datasets on the map there was an error I introduced earlier).

**Pull request merged**